provider/lxd/server.go

```diff
@@ -52,6 +52,7 @@ type Server interface {
 	CreateProfileWithConfig(string, map[string]string) error
 	GetProfile(string) (*lxdapi.Profile, string, error)
 	HasProfile(string) (bool, error)
+	CreateProfile(post lxdapi.ProfilesPost) (err error)
```

Comment:
Help needed here pls: I ran mock gen after adding this, however I had to edit by hand, since it created a circular reference and wouldn't work.  Not sure why.  MockServer.CreateProfile() and MockServerMockRecorder.CreateProfile() work find.

Reply:
:eyes:

Reply:
So I ran `go generate .` from inside `provider/lxd` and although it rearranged the mock file slightly, it still allowed me to run `go test ...` fine.

Two questions: 
 1. what gomock have you installed atm
 1. can you put up the file that it did generate?

Reply:
updating mockgen, as recommended, worked.